packages/kbn-search-connectors/lib/create_connector.ts

```diff
@@ -23,19 +23,46 @@ export const createConnector = async (
     name?: string;
     pipeline?: IngestPipelineParams;
     serviceType?: string | null;
-    instant_response?: boolean;
```

Comment:
We have `immediate` refresh policy by default: https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/ConnectorIndexService.java#L133